server/datastore/mysql/software_installers.go

```diff
@@ -3097,7 +3097,7 @@ func (ds *Datastore) getIncludedHostUUIDMapForSoftware(ctx context.Context, tx s
 	filter := fmt.Sprintf(labelScopedFilter, swType)
 	stmt := fmt.Sprintf(`SELECT
 		h.uuid AS uuid,
-		ad.applied_policy_id AS applied_policy_id
+		COALESCE(ad.applied_policy_id, '') AS applied_policy_id
```

Comment:
if we're doing this, should we change `AppliedPolicyID` in the results struct below to be a `string` instead of a pointer?